lib/logger.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from enum import Enum
+
+class LogLevel(Enum):
+    DEBUG = "DEBUG"
+    MESSAGE = "MESSAGE"
+    WARNING = "WARNING"
+    ERROR = "ERROR"
+
+DATE_FORMAT = "%b %d %H:%M:%S"
+
+def _log(level, message, end):
+    time = datetime.now()
+    print("{} {}: {}".format(time.strftime(DATE_FORMAT), level.value, message), end=end)
+
+def debug(message, end='\n'):
+    _log(LogLevel.DEBUG, message, end)
```

Comment:
It would be interesting to add a variable in the project config to specify the level of verbosity. More particularly to hide the debug logs.

Reply:
Yes I thought about it. I'm looking at how to do it.

Reply:
Take a look at: https://github.com/xcp-ng/xcp-ng-tests/blob/master/lib/config.py ;)

Reply:
And: https://github.com/xcp-ng/xcp-ng-tests/blob/master/conftest.py#L67